data/content/about.md

```diff
@@ -1,3 +1,51 @@
-Free and guided workshops in Rust – a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety.
+RustBridge is a workshop focused on getting underrepresented people with
+a background in another programming language to learn Rust and join the
```

Comment:
Yeah, we struggled at that on multiple levels, it's also hard to come up with teasing examples right from the get-go for beginners.